src/frameworks/foundation/ns_string.rs

```diff
@@ -591,7 +591,7 @@ pub fn handle_constant_string(mem: &mut Mem, objc: &mut ObjC, constant_str: id)
         bytes,
         length,
     } = mem.read(constant_str.cast());
-    assert!(flags == 0x7C8); // no idea what this means
+    assert!(flags == 0x7C8); // see: https://lists.llvm.org/pipermail/cfe-dev/2008-August/002518.html
```

Comment:
Aha, I suspected it was some CFType thing but knowing the details is great, thanks!